101-mongodb-sharded-on-centos/nested/config-primary-resources.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
these templates for the vms are largely identical - let's combine the 5 (router, config and replica) into a single template and use parameters to set the behavior.

I'm only going to leave the comments on the superset.

Reply:
I have combined the config and replica templates. I'd like to keep router, config and replica templates separate so I can make changes easily in the future.